### DISCUSSION TITLE
FR: possibility to pass the prompt as a GET request parameter

### DISCUSSION THREAD

**Discussion by akuemin-adc** [danny-avila/LibreChat#5113] [Feature Requests & Suggestions]
Have possibility to pass the prompt as a GET request parameter like in Perplexity: https://www.perplexity.ai/search/new?q=Hello%21+this+is+a+prompt+in+GET+request

Use Case: it would allow to be the last step in automation flow. e.g. in excel having several links to articles that may or may not be relevant for my company, then be able to dynamically concatenate a prompt text and make a link to it in adjacent excel cell.

EDIT: Ideally the prompt would be send to the system directly, without manual intervention

![image (8)](https://github.com/user-attachments/assets/ff228207-1907-4514-b8c4-1e6829f62437)

**Comment by danny-avila** [comment]
It’s already implemented. https://github.com/danny-avila/LibreChat/issues/1343

**Reply by akuemin-adc** [reply]
That's great! :) but using ?prompt=[text] it only fills in the prompt in the input box. One still need to click on the arrow or press enter to actually send the prompt to the system.

Is there way to send the query to the system rather than just pre-filling it to the input box? e.g. a parameter &target=system rather than &target=inputbox.

Is there a documented list of all GET parameters available in librechat? unfortunately I didn't find it with the search function in the librechat docs

Thanks!
